src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -27,6 +27,12 @@
  *
  */
 public class LambdaHandler extends AbstractExpressionHandler {
+    /**
+     * Checks whether the lambda is correctly indented, this variable get its value from checking
+     * the lambda handler's indentation, and it is being used in aligning the lambda's children.
+     * A true value depicts lambda is correctly aligned without giving any errors.
+     */
+    private boolean isIdealIndent;
```

Comment:
`isAcceptableIndent`

Reply:
`isAcceptableIndent` is already the name of a method. Thus renaming it is creating pmd violation that no same name for method and variable is allowed.
Though I have changed it something more accurate about what it actually does.
This variable checks if the lamda expression is indented properly, and thus upon `yes/no` it states its children expected indent.